docs/diagnostics/PX1007.md

```diff
@@ -8,7 +8,10 @@ This document describes the PX1007 diagnostic.
 | PX1007 | A DAC, a DAC extension, and public DAC properties should have a description in the `summary` XML tag. | Warning | Available | 
```

Comment:
The overview should be fixed to mention inheritdoc tag. 
The same applies to the Summary.md file